Thanks @sdbondi - 2 last errors in wallet_ffi 

```
error[E0432]: unresolved import `tari_core::transactions::transaction_protocol::TxId`
  --> base_layer/wallet_ffi/src/callback_handler.rs:55:5
   |
55 | use tari_core::transactions::transaction_protocol::TxId;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `TxId` in `transactions::transaction_protocol`

error[E0432]: unresolved import `tari_core::transactions::transaction_protocol::TxId`
   --> base_layer/wallet_ffi/src/lib.rs:200:5
    |
200 | use tari_core::transactions::transaction_protocol::TxId;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `TxId` in `transactions::transaction_protocol`
```

--- a/base_layer/wallet_ffi/src/callback_handler.rs
+++ b/base_layer/wallet_ffi/src/callback_handler.rs
@@ -52,7 +52,6 @@ use log::*;
 use tari_common_types::transaction::TxId;
 use tari_comms::types::CommsPublicKey;
 use tari_comms_dht::event::{DhtEvent, DhtEventReceiver};
-use tari_core::transactions::transaction_protocol::TxId;
 use tari_shutdown::ShutdownSignal;
 use tari_wallet::{
     output_manager_service::handle::{OutputManagerEvent, OutputManagerEventReceiver},

--- a/base_layer/wallet_ffi/src/lib.rs
+++ b/base_layer/wallet_ffi/src/lib.rs
@@ -143,10 +143,16 @@ use tari_crypto::{
 use tari_utilities::{hex, hex::Hex};
 use tokio::runtime::Runtime;
 
+use crate::{
+    callback_handler::CallbackHandler,
+    enums::SeedWordPushResult,
+    error::{InterfaceError, TransactionError},
+    tasks::recovery_event_monitoring,
+};
 use error::LibWalletError;
 use tari_common_types::{
     emoji::{emoji_set, EmojiId, EmojiIdError},
-    transaction::{TransactionDirection, TransactionStatus},
+    transaction::{TransactionDirection, TransactionStatus, TxId},
     types::{ComSignature, PublicKey},
 };
 use tari_comms::{
@@ -186,18 +192,10 @@ use tari_wallet::{
     WalletSqlite,
 };
 
-use crate::{
-    callback_handler::CallbackHandler,
-    enums::SeedWordPushResult,
-    error::{InterfaceError, TransactionError},
-    tasks::recovery_event_monitoring,
-};
-
 mod callback_handler;
 mod enums;
 mod error;
 mod tasks;
-use tari_core::transactions::transaction_protocol::TxId;
 
 const LOG_TARGET: &str = "wallet_ffi";
 

